tests/test_projects.py

```diff
@@ -62,17 +95,18 @@ def test_project_serialization():
     assert 'created' in context
 
 
-def test_project_metadata_compatibility():
+@pytest.mark.parametrize("project_meta,version", [(PROJECT_V1, '1'), (PROJECT_V2, '2')])
+def test_project_metadata_compatibility(project_meta, version):
```

Comment:
Maybe we should add a test where we assert for translation of schema.

Reply:
I started to do that but then realized that if the translation doesn't happen, `name` is set to `None` so this test fails.

Reply:
I agree though that maybe an explicit test would be better - do you have a suggestion?

Reply:
https://gist.github.com/jsam/3329f961d8062d6d917eddbb5dc45cd2